src/components/card/card.component.jsx

```diff
@@ -0,0 +1,32 @@
+import "./card.style.css";
+function Card({ title, data, size }) {
+    return (
+        <div className={`${size} card`}>
+            <div className="card-text-cont">
+                <p className="card-title">{title}</p>
+                {Object.keys(data).length > 1 ? (
```

Comment:
for better readability we can create a variable called for example `isDataEmpty` and use it to render this conditionally